base/rational.jl

```diff
@@ -147,9 +147,9 @@ end
 Rational(x::Rational) = x
 
 Bool(x::Rational) = x==0 ? false : x==1 ? true :
-    throw(InexactError(:Bool, Bool, x)) # to resolve ambiguity
+    throw(InexactError(:convert, Bool, x)) # to resolve ambiguity
 (::Type{T})(x::Rational) where {T<:Integer} = (isinteger(x) ? convert(T, x.num)::T :
-    throw(InexactError(nameof(T), T, x)))
+    throw(InexactError(:convert, T, x)))
```

Comment:
We could put a sentinel value here like `Symbol("")`, in which case the printer for InexactError knows to show the type as the called object.